packages/formation/sass/base/_b-variables.scss

```diff
@@ -105,7 +105,7 @@ $color-primary-alt-darkest:  #046b99;
 $color-green-darker:         #195c27;
 $color-va-accent:            #988530; // New gold "metal" acccent
 $color-gold:                 #fdb81e;
-$color-warning-message:      #fac922; // Used for Disability benefits
```

Comment:
This variable is being used in vets-website still. I'm wondering what would be the best way to do this changeover so they aren't interrupted.  

Maybe we need to keep this variable here for now and add `$color-feedback-warning-background` too? Then update vets-website after it's available?

![Screenshot 2023-10-20 at 10 15 54 AM](https://github.com/department-of-veterans-affairs/veteran-facing-services-tools/assets/872479/7efd42c4-c4ca-48f6-8b29-45e19c03cba1)

Reply:
For what it's worth, I'm changing this in vets-website right now, too, so there won't be any interruption. That PR should be up in the next few minutes.

Reply:
This repo is not directly linked to vets-website so just merging the PR won't make it available there. A new release of Formation will need to be created with this change and vets-website updated with the new version too (in your PR there would probably be easiest).

Before this PR is merged, you'll need to update the `package.json` version number and then follow [step 3 in the docs for publishing Formation](https://department-of-veterans-affairs.github.io/veteran-facing-services-tools/getting-started/common-tasks/updating-formation/).

Reply:
Hmm, looks like you were right, I can't make the change in vets-website until after I've made that variable available in formation. So we either have to publish the new version of formation and then bring it in at the same time as the variable name change, or make both variables available in formation, bring it in and change vets-website, and then remove it from formation and publish another release. My preference would be to just publish a new formation release once, personally.

Reply:
Version bumped, I believe the PR needs to be merged into master before I can publish.